src/events.rs

```diff
@@ -312,3 +312,11 @@ pub enum VirtualKeyCode {
     WebStop,
     Yen,
 }
+
+#[derive(Default, Debug, Clone, Copy)]
+pub struct Mods {
+  pub shift: bool,
+  pub ctrl: bool,
+  pub alt: bool,
+  pub supr: bool
```

Comment:
Shame that `super` is a reserved keyword. I find `supr` quite a confusing name...

an other common name for this key (in linux world at least) is `mod4` (`alt` is also `mod1`, I don't know what are 2 and 3), but I don't think it's a clear name either...

`super_` would do the work, but is ugly ... I'm out of ideas here... At least it needs to be clearly documented what key it represents (telling in the docs that it is the "windows key" on most keyboards)